cirq-core/cirq/neutral_atoms/convert_to_neutral_atom_gates.py

```diff
@@ -92,32 +94,8 @@ def optimization_at(
 
 
 def is_native_neutral_atom_op(operation: ops.Operation) -> bool:
-    if isinstance(operation, (ops.GateOperation, ops.ParallelGateOperation)):
-        gate = operation.gate
-        if isinstance(gate, ops.ParallelGate):
-            gate = gate.sub_gate
-        return is_native_neutral_atom_gate(gate)
-    return False
+    return operation in neutral_atom_devices.neutral_atom_gateset()  # coverage: ignore
```

Comment:
Why is this coverage ignore?  Can we add a quick unit test for it?  It seems straight forward.  Same below.

Reply:
Added coverage tests.